source/client/remote_process_impl.cc

```diff
@@ -0,0 +1,58 @@
+#include "client/remote_process_impl.h"
+
+#include <grpc++/grpc++.h>
+
+#include <memory>
+
+#include "nighthawk/client/output_collector.h"
+
+#include "api/client/options.pb.h"
+#include "api/client/output.pb.h"
+
+#include "common/uri_impl.h"
+
+#include "client/options_impl.h"
+
+namespace Nighthawk {
+namespace Client {
+
+RemoteProcessImpl::RemoteProcessImpl(const Options& options,
+                                     nighthawk::client::NighthawkService::Stub& stub)
+    : options_(options), stub_(stub) {}
+
+bool RemoteProcessImpl::run(OutputCollector& collector) {
+  nighthawk::client::ExecutionRequest request;
+  nighthawk::client::ExecutionResponse response;
+  grpc::ClientContext context;
+  auto r = stub_.ExecutionStream(&context);
```

Comment:
(nit) Can we give this variable a slightly longer; more readable name?

Reply:
Done in 5f7fa78